packages/contracts/contracts/registry/AragonPluginRegistry.sol

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "../core/component/ERC165Registry.sol";
+import "../core/IDAO.sol";
+import "../APM/IPluginRepo.sol";
+
+/// @title Register plugin
+/// @author Sarkawt Noori - Aragon Association - 2022
+/// @notice This contract provides the possiblity to register a plugin pluginRepo by a unique address.
+contract AragonPluginRegistry is ERC165Registry {
+    /// @notice Emitted if a new PluginRepo is registered
+    /// @param name The name of the PluginRepo
+    /// @param pluginRepo The address of the PluginRepo
+    event NewPluginRepo(string name, address pluginRepo);
+
+    /// @notice Initializes the contract
+    /// @param _dao the managing DAO address
+    function initialize(IDAO _dao) public initializer {
+        bytes4 pluginRepoInterfaceId = type(IPluginRepo).interfaceId;
+        __ERC165Registry_init(_dao, pluginRepoInterfaceId);
+    }
+
+    /// @notice Registers a PluginRepo
+    /// @param name The name of the PluginRepo
+    /// @param registrant The address of the PluginRepo contract
+    function register(string calldata name, address registrant) external {
+        _register(registrant);
+
+        emit NewPluginRepo(name, registrant);
```

Comment:
IMO, an event named `NewPluginRepo` should be emitted in the context of creation (i.e. in `RepoFactory`, where the `PluginRepo` is created via `createProxy` rather than in the registry, where it is just registered.

Reply:
PluginRepoRegistered converted to PluginRepoRegistered